### PR TITLE
Small changes on refactor APIKeyLastUseCache get

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Follow #4201 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/ops/apikeylastusedtimeupdater/apikeylastusedtimeupdater.go [hunk 1]
```diff
@@ -98,23 +98,25 @@ func (c *APIKeyLastUsedTimeUpdater) updateAPIKeyLastUsedTime(ctx context.Context
...
-			c.logger.Error("failed to update last used time",
+			c.logger.Error("failed to fetch last used time from cache",
 				zap.String("id", apiKey.Id),
 				zap.Error(err),
 			)
 			continue
 		}
 
-		lastUsedTime, err := bytes2int64(lastUsedTimeByte.([]byte))
+		lastUsedTime, err := strconv.ParseInt(string(cachedLastUse.([]byte)), 10, 64)
```
Comment (knanao) [inline pkg/app/ops/apikeylastusedtimeupdater/apikeylastusedtimeupdater.go:110]: nits
```suggestion
                s := string(cachedLastUse.([]byte))
		lastUsedTime, err := strconv.ParseInt(s, 10, 64)
```

File: pkg/app/ops/apikeylastusedtimeupdater/apikeylastusedtimeupdater.go [hunk 2]
```diff
@@ -98,23 +98,25 @@ func (c *APIKeyLastUsedTimeUpdater) updateAPIKeyLastUsedTime(ctx context.Context
...
 		if err != nil {
-			c.logger.Error("failed to update last used time",
+			c.logger.Error("failed to fetch last used time from cache",
```
Comment (knanao) [inline pkg/app/ops/apikeylastusedtimeupdater/apikeylastusedtimeupdater.go:112]:
```suggestion
			c.logger.Error("failed to parse last used time from cache",
```

File: pkg/app/server/grpcapi/web_api.go [hunk 3]
```diff
@@ -1528,14 +1528,16 @@ func (a *WebAPI) ListAPIKeys(ctx context.Context, req *webservice.ListAPIKeysReq
...
-		// Get LastUsedTIme from Redis
-		var lastUsedAtFromCache int64 = 0
-		if lastUsedAtFromCacheByte, error := a.apiKeyLastUsedStore.Get(apiKeys[i].Id); error == nil {
-			lastUsedAtFromCache = bytes2int64(lastUsedAtFromCacheByte.([]byte))
+		var lastUsedAt int64
+		if cachedLastUse, err := a.apiKeyLastUsedStore.Get(apiKeys[i].Id); err == nil {
+			lastUsedAt, err = strconv.ParseInt(string(cachedLastUse.([]byte)), 10, 64)
+			if err != nil {
+				a.logger.Error("failed to get last used time from cache", zap.Error(err))
+			}
```
Comment (knanao) [inline pkg/app/server/grpcapi/web_api.go:1536]: We should check the 1532 line's error?

Comment (khanhtc1202) [reply pkg/app/server/grpcapi/web_api.go]: Sure do, I updated this part 🙆‍♀️

File: pkg/app/server/grpcapi/web_api.go [hunk 4]
```diff
@@ -1528,15 +1528,29 @@ func (a *WebAPI) ListAPIKeys(ctx context.Context, req *webservice.ListAPIKeysReq
...
+				zap.Error(err),
+			)
+			continue
 		}
 
-		if lastUsedAtFromCache > apiKeys[i].LastUsedAt {
-			apiKeys[i].LastUsedAt = lastUsedAtFromCache
+		var lastUsedAt int64
+		s := string(cachedLastUse.([]byte))
+		lastUsedAt, err = strconv.ParseInt(s, 10, 64)
```
Comment (knanao) [inline pkg/app/server/grpcapi/web_api.go:1542]:
```suggestion
		s := string(cachedLastUse.([]byte))
		lastUsedAt, err := strconv.ParseInt(s, 10, 64)
```

Comment (khanhtc1202) [reply pkg/app/server/grpcapi/web_api.go:1542]: 確かにw

**Review by knanao — APPROVED** [review]
Here you go!

**Pull request merged**